lib/request.js

```diff
@@ -115,7 +115,8 @@ function buildRequestWithTrustProxy (R, trustProxy) {
   Object.defineProperties(_Request.prototype, {
     ip: {
       get () {
-        return proxyAddr(this.raw, proxyFn)
+        const addrs = proxyAddr.all(this.raw, proxyFn)
+        return addrs[addrs.length - 1]
       }
     },
```

Comment:
Just leaving a note here that this won't slow down the performance if only one IP exists, as `proxyAddr` itself also calls `proxyAddr.all`, and returns the last element of the returned array, so it does the same thing